docs/UserGuide.adoc

```diff
@@ -120,6 +124,78 @@ Original name was: Laksa`
 Original cost was: $10.50`
 // end::update[]
 
+//tag::tag[]
+[[TagTag]]
+
+=== Tag transaction: `tag`
+
+Format: `tag i/INDEX t/TAG_NAME`
+
+****
+* Tags the transaction at the specified `INDEX` with `TAG_NAME`.
+* `INDEX` refers to the index number currently shown in the UI (after filtering, if any).
+* `INDEX` *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+*   `tag i/8 t/Debt`
+
+Expected output:
+
+*   `Entry tagged: 8. Student Loan ($12000.00) [Debt]`
+
+// end::tag[]
+
+// tag::untag[]
+
+=== Tag transaction: `untag`
+
+Format: `untag i/INDEX t/TAG_NAME`
+
+****
+* Untags `TAG_NAME` from the transaction at the specified `INDEX`.
+* `INDEX` refers to the index number currently shown in the UI (after filtering, if any).
+* `INDEX` *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+*   `untag i/7 t/Important`
+
+Expected output:
+
+*   `Entry untagged: 7. DoTA Arcanas ($49.00) [Game]`
+
+// end::untag[]
+
+// tag::convert[]
+
+=== Convert currency: `convert`
+
+Format: `convert f/ORIGIN_CURRENCY t/TARGET_CURRENCY (v/VALUE|i/INDEX)`
```

Comment:
Should not use prefix `t/` as that is used for `TAG` already, maybe use `from/` and `to/` or something similar? Also could consider using `c/` instead of `v/` since it is regarding money and `c/` is used in `Transaction`.

Reply:
I have changed it to `c/TARGET_CURRENCY` only and removed ORIGIN_CURRENCY as it was actually redundant.

Reply:
I meant that you should use `c/` for the amount instead of the currency name, since `/c` is already used for specifying amount in adding `Transaction`.

Perhaps: 
`convert currency/TARGET_CURRENCY (c/COST | i/INDEX)`

Reply:
I have changed the command format to `convert to/TARGET_CURRENCY (c/COST | i/INDEX)`.

Reply:
Great! Thank you 😁